src/TicketingSystem.BusinessLogic/Dtos/TicketDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using TicketingSystem.BusinessLogic.Enums;
```

Comment:
Looks, like copy paste - namespace should be Dtos?

Reply:
See no issues here, System namespace is required because of using DateTimeOffset, BusinessLogic.Enums is required because of BLL enums used below

Reply:
Enums were moved into a separate Common project to use them without mapping, namespace was updated